src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -4872,6 +5407,9 @@ access(const char *path, int mode)
 
 out_err:
 	FREE(parent_dir);
+	/* dfs_access returns EIO sometimes. Give another chance. */
```

Comment:
dfs_access should not return EIO without good reason, do you know why this is?  This seems like it would warrant a ticket, having the ticket number here and logging an error if it occours.

Reply:
I do not have the details now. I need to reproduce the issue of EIO in daos build test and get more information later, then file a ticket. Thank you!

Reply:
I could not reproduce the EIO issue yesterday locally, so I removed the retry through dfuse. daos build test passed in CI.